src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml

```diff
@@ -26,29 +26,29 @@ limitations under the License.
     <!-- TODO: Add feature map once it is supported -->
     <description>This cluster provides an interface for controlling the current Channel on a device.</description>
     <attribute side="server" code="0x0000" define="CHANNEL_LIST" type="ARRAY" entryType="ChannelInfo" length="254" writable="false" optional="false">channel list</attribute>
-    <attribute side="server" code="0x0001" define="CHANNEL_LINEUP" type="ChannelLineupInfo" writable="true" optional="false">channel lineup</attribute>
+    <attribute side="server" code="0x0001" define="CHANNEL_LINEUP" type="LineupInfo" writable="true" optional="false">channel lineup</attribute>
     <attribute side="server" code="0x0002" define="CHANNEL_CURRENT_CHANNEL" type="ChannelInfo" writable="true" optional="false">current channel</attribute>
 
-    <command source="client" code="0x00" name="ChangeChannel" optional="false">
+    <command source="client" code="0x00" name="ChangeChannelRequest" response="ChangeChannelResponse" optional="false">
```

Comment:
Please file a spec issue about it not being clear which "ChangeChannel" it means for the response.